### PR TITLE
ICU-21958 Update Collation Demo index.html for ICU72.1

### PR DESCRIPTION
**Pull request by sven-oly**
Updates with new CLDR and ICU data

### CODE REVIEW

File: webdemo/collation/index.html [hunk 1]
```diff
@@ -66,19 +66,20 @@ <h1><a href="https://icu.unicode.org/">ICU</a> Collation Demo</h1>
 <option value="en_US_POSIX">en-US-u-va-posix (type=standard):   American English (Computer, Standard Sort Order)</option>
 <option value="eo">eo (type=standard):   Esperanto (Standard Sort Order)</option>
 <option value="es">es (type=standard):   Spanish (Standard Sort Order)</option>
-<option value="es@collation=search">es-u-co-search (normalization=on):   Spanish (General-Purpose Search)</option>
+<option value="es@collation=search">es-u-co-search (normalization=on):   Spanish (General-Purpose Search)</option> meld webdemo/collation/index.html ../available-collators.txt
```
Comment (markusicu) [inline webdemo/collation/index.html:69]:
```suggestion
<option value="es@collation=search">es-u-co-search (normalization=on):   Spanish (General-Purpose Search)</option>
```

looks like a "paste" gone astray

**Review by markusicu — COMMENTED** [review]
one snafu, otherwise lgtm

File: webdemo/collation/index.html (hunk 1 above)
Comment (sven-oly) [reply webdemo/collation/index.html]: Done. Thanks for the careful review. PTAL.

**Review by markusicu — APPROVED** [review]
lgtm tnx

Please **squash**-and-merge, and please make sure that the squash commit message starts with the ICU ticket number. (Especially since the second commit here does not have that.)

**Pull request merged**